website/pages/docs/configuration/telemetry.mdx

```diff
@@ -37,6 +37,22 @@ The following options are available on all telemetry configurations.
 - `enable_hostname_label` `(bool: false)` - Specifies if all metric values should
   contain the `host` label with the local hostname. It is recommended to enable
   `disable_hostname` if this option is used.
+- `lease_metrics_epsilon` `(string: "1hr")` - Specifies the time difference between collections
+  of leases by expiry time. For example, for the default value of 1 hour, the `vault.expire.leases.by_expiration`
```

Comment:
Hm, "time difference between collection of leases" makes it sounds like it is the interval at which the collection is done, not the interval of the underlying data.

How about something like "the size of the bucket used to measure future lease expiration?"  Or "the granularity of measurement for lease expiration"?